core/src/App.html

```diff
@@ -1088,7 +1111,7 @@
 
     EventListenerHelpers.addEventListener('popstate', async e => {
       const alertQueue = alerts;
-      if (!alertQueue || !alertQueue.length) return;
+      if (!alertQueue || !alertQueue.length > 0) return;
```

Comment:
Wrong check... in this context it always leads to the correct result, but be aware that 
!x > 0 is not the same as !(x > 0) for negative x!